bin/journey-test.sh

```diff
@@ -269,18 +269,24 @@ main() {
 
   clean "${build_dir}"
 
-  # Make a local version of trackler that includes the source from this repo.
-  git_clone "x-api" "${xapi_home}"
-  git_clone "trackler" "${trackler_home}"
-  assert_ruby_installed "${trackler_home}"
+  # Download everything we need in parallel
+  git_clone "x-api" "${xapi_home}" &
+  git_clone "trackler" "${trackler_home}" &
+  download_exercism_cli $(get_operating_system) $(get_cpu_architecture) "${exercism_home}" &
+  wait
```

Comment:
https://en.wikipedia.org/wiki/Wait_(command)
Basically it *waits* for the commands running in the background to finish before continuing.